### ISSUE TITLE
Composer: Your requirements could not be resolved to an installable set of packages

### ISSUE THREAD

**Issue by kedniko** [messente/messente-api-php#17]
**Error**

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires messente/messente-api-php ^2.0 -> satisfiable by messente/messente-api-php[2.0.0].
    - messente/messente-api-php 2.0.0 requires guzzlehttp/psr7 ^1.8 -> found guzzlehttp/psr7[1.8.0, ..., 1.x-dev] but the package is fixed to 2.4.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require messente/messente-api-php:*" to figure out if any version is installable, or "composer require messente/messente-api-php:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Step to reproduce the error**

`composer global require laravel/installer`

`laravel new example-app`

`cd example-app`

`composer require messente/messente-api-php`

**System**
OS: Windows 10
PHP version: 8.2.0
Composer: 2.4.4

The error is similar to https://github.com/messente/messente-api-php/issues/13

**Comment by glintik** [comment]
Looks like this issue to be fixed by #16.

**Comment by kedniko** [comment]
Thanks @glintik

Solution:
- delete `composer.lock`
- in `composer.json` add
```
{
  "require": {
    "messente/messente-api-php": "dev-master#178699a" // <-- latest commit id
  }
}
```
- `composer update`
